### PR TITLE
Use links prop, simplify a bit

### PR DESCRIPTION
**Pull request by Daimona**

### CODE REVIEW

File: orphanize.php [hunk 1]
```diff
@@ -98,111 +98,72 @@
...
-			// just the page title
-			$title = explode( '|', $wlink )[0];
-
-			// normalize titles
-			$title = str_replace( '_', ' ', ucfirst( $title ) );
-
-			// append
-			$titles_to_be_orphanized[] = $title;
-		}
+$titles_to_be_orphanized = $links->query->pages[0]->links ?? null;
```
Comment (Daimona) [inline orphanize.php]: Not [0] but reset().

**Comment by valerio-bozzolan** [comment]
(Just please keep PHP 5.6 backward compatibility, avoid `??`)

File: orphanize.php [hunk 2]
```diff
@@ -158,8 +158,8 @@ function( $t ) {
 	];
 
 	// limit to certain namespaces
-	if( isset( $NS ) ) {
-		$linksto_args[ 'lhnamespace' ] = $NS;
+	if( $NS !== null ) {
+		$linksto_args[ 'lhnamespace' ] = implode( '|', $NS );
```
Comment (valerio-bozzolan) [inline orphanize.php]: Note that it was not a problem a pipe-separated string straight on in `lhnamespace`. We can safetly avoid imploding this.

**Comment by valerio-bozzolan** [comment]
Here we are.

About 69db566c1640e19e0ebcf50ef2ec18010cfee57e, I have to say that the reason below the older scraping way was also because I was thinking about adding support for a small placeholder such as `<!-- edit below this line -->` in order to distinguish the involved links from some instructions eventually affixed on the top of the page (eventually full of other links to be not involved in the process, you know). Well, anyway your cleaner solution can be used toghether with a separated page of instructions that simply transcludes the lists at the bottom and that so it's even more simpler and even more error-proof.

TL;DR Thank you for your different approach.

**Comment by Daimona** [comment]
> can be used toghether with a separated page of instructions that simply transcludes the lists at the bottom and that so it's even more simpler and even more error-proof.

This.
